### DISCUSSION TITLE
v19.23.3-4.0.1 - YTLitePlus

### DISCUSSION THREAD

**Discussion by Balackburn** [YTLitePlus/YTLitePlus#178] [Announcements]
# ⚠️ Must Read ⚠️
- The .ipa file is no longer shared on Github due to a [Google DMCA notice](https://www.reddit.com/r/jailbreak/comments/1b86tiz/balackburn_here_maintainer_of_ytliteplus_ive_just/) 
- You need to build it yourself. See [Wiki-Building](https://github.com/Balackburn/YTLitePlus/wiki/Building)

# YTLitePlus Release Information

Current YouTube IPA: `19.23.3`  
Current YTLite Version: `4.0.1`  
Current App Compatibility: `iOS/iPadOS 15.0` or later

**RELEASE 19.23.3-4.0.1:** (2024-06-10)

- Updated Youtube from `19.22.6` to `19.23.3`
- Updated submodules
- Fixed compiling

  
### Issues in this Release:
- LowContrastMode does not work *(Compatible only with YouTube v17.33.2-v17.38.10)*

### Catbox
`pmsw97.ipa`

<hr /><em>This discussion was created from the release <a href='https://github.com/YTLitePlus/YTLitePlus/releases/tag/v19.23.3-4.0.1'>v19.23.3-4.0.1 - YTLitePlus</a>.</em>

**Comment by Joshua-18** [comment]
The new button for shorts crashes for me when i tap on it

**Reply by backstab5983** [reply]
Works for me.

**Reply by Joshua-18** [reply]
Weird cause I have other devices and they all crash as soon as I press it, same thing with a friend of mine.

**Reply by Balackburn** [reply]
> The new button for shorts crashes for me when i tap on it

Device, ios version, how did you sideload?

**Reply by Joshua-18** [reply]
> > The new button for shorts crashes for me when i tap on it
> 
> Device, ios version, how did you sideload?

Iphone 14 pro max , 16.6.1, trollstore.

**Reply by Balackburn** [reply]
> > > The new button for shorts crashes for me when i tap on it
> 
> > 
> 
> > Device, ios version, how did you sideload?
> 
> 
> 
> Iphone 14 pro max , 16.6.1, trollstore. 

Could you try with sideloadly or Altstore this may come from trollstore (not sure)

**Reply by Joshua-18** [reply]
I used esign with a cert and it doesn’t crash, you might be right it could be trollstore.

**Comment by TV365-VN** [comment]
Can you build YTPlus on Android

**Reply by Balackburn** [reply]
No and it will never be possible, use Revanced instead

**Reply by TV365-VN** [reply]
Bạn có link tải Revanced không, bạn có thể giới thiệu giúp cho tôi link tải an toàn của Revanced

**Reply by y4wnn** [reply]
[site](https://revanced.app/)
[github](https://github.com/ReVanced/revanced-manager/releases)
[instructions/hướng dẫn sd](https://github.com/ReVanced/revanced-manager/tree/main/docs)

revanced manager is an app that lets you pick patches you want for certain supported apks/revanced manager là ứng dụng giúp bạn tự thêm "patch" mình muốn vào một số apk đc hỗ trợ

**Comment by HeKsT8695** [comment]


**Reply by Balackburn** [reply]
> My Homepage tab "All" is bugged. Theres only a black screen, when I scroll down a bit some videos show up, but disappear again. In tabs like "Gaming" or "New For You" everything is fine. I use iPhone 12 with iOS 17.5.1

What are your settings? (Please screen record all the settings from YTLitePlus and YouTube Plus section)

**Reply by HeKsT8695** [reply]
I have downloaded the latest version now and everything seems to work fine now. I was using a way older version before, my bad

**Comment by shoperfy** [comment]
I forked the repo and ran the actions with an ipa for 19.23.3 and for some reason in releases I get 19.22.6 instead of the latest version. Does that mean that the ipa was 19.22.6?

**Reply by Balackburn** [reply]
> I forked the repo and ran the actions with an ipa for 19.23.3 and for some reason in releases I get 19.22.6 instead of the latest version. Does that mean that the ipa was 19.22.6?

The version is extracted from info.plist, your .ipa is probably mislabeled as 19.23.3 but is 19.22.6, this happens sometimes with .ipa from decrypt.day

**Comment by HeKsT8695** [comment]
When I download through appdb or maplesigner I don't get any notifications at all. Is this a thing when you download third party ipa?

**Reply by Balackburn** [reply]
Yes

**Comment by HeKsT8695** [comment]
Is it possible to add a feature, so you can watch videos that are age restricted, or only available for special channel subscriptions?

**Comment by helixxger** [comment]
Network playback is not working

i reainstalled the app, but I cannot search for network devices to play videos on :(

**Comment by bpGusar** [comment]
If in setting I set video quality over WiFi to Best then open 2K video I see only 1080p if using classic quality menu, but if I open 4K HDR video then in quality menu 2K quality is available. Is this a bug?

**Reply by backstab5983** [reply]
Can't reproduce, works for me

**Reply by bpGusar** [reply]
When you watch a MKBHD video, do you see something higher than 1080p in the quality settings? because I don't

**Reply by backstab5983** [reply]
> When you watch a MKBHD video, do you see something higher than 1080p in the quality settings? because I don't

Yes, I see up to 2160p. Check you settings ;)

**Reply by bpGusar** [reply]
![IMG_1971](https://github.com/YTLitePlus/YTLitePlus/assets/84081188/7c5a44e7-5f8e-408d-a171-e635ff39d749)
![IMG_1970](https://github.com/YTLitePlus/YTLitePlus/assets/84081188/6a7aa946-2a01-4e6b-92bb-15df54b35e02)

How is this possible? The settings say Better/Best (I don't know how it's named in English) quality, but the video is still a maximum of 1080p

**Reply by backstab5983** [reply]
VP9 codec has to be enabled.

**Reply by bpGusar** [reply]
That is sooo stupid I swear. How should I know this?  ))
Thanks, now 4k is available for me

**Reply by shoperfy** [reply]
you should know it as whenever you get a new piece of tech or app you always go through the settings lol

**Comment by abheizthe** [comment]
Does anyone having this issue?
![0D1209D0-1EA1-4EEC-B7A6-C0E4902E43F0](https://github.com/YTLitePlus/YTLitePlus/assets/54247318/3280debc-d6d7-4ccb-b3e0-063233945f19)

the image on "posts" cannot be opened in full size when click on it
I forgot from which version this issue but until now it's still persist

**Comment by phattrance** [comment]
I am not able to AirPlay any videos to my Apple  tv
 via this version. It asked for permission To local network access, but it is already allowed in the settings.
How can I fix this?

![IMG_7924](https://github.com/user-attachments/assets/e82bdc08-5471-46c2-8f9e-e119e3c42a6c)
![IMG_7925](https://github.com/user-attachments/assets/95b63979-2737-4b80-834a-15fb7af0caa0)

**Reply by FrankTheTank07** [reply]
Where did you get your IPA from? Your YouTube version is a little outdated and this issue was recently fixed. 

https://github.com/YTLitePlus/YTLitePlus/issues/211#

Also what model iPhone are you using and what software version?

**Reply by phattrance** [reply]
> Where did you get your IPA from? Your YouTube version is a little outdated and this issue was recently fixed.
> 
> #211
> 
> Also what model iPhone are you using and what software version?

I downloaded it from here: https://altsource.by.lao.sb/browse/?source=https%3A%2F%2Fraw.githubusercontent.com%2FBalackburn%2FYTLitePlusAltstore%2Fmain%2Fapps.json

im not able to find the latest IPA to sign, if you have a link to a newer IPA i can try installing it and test it.

Using iPhone 13 pro max with latest iOS firmware (17.5.1)

**Comment by geeflaxx** [comment]
With sidestore it will be crashed all the time ( iPad: 17.5.1 )
